pulp_rpm/app/modulemd.py

```diff
@@ -0,0 +1,86 @@
+import json
+import tempfile
+
+from django.db.utils import IntegrityError
+from pulpcore.app.models.content import Artifact
+from pulp_rpm.app.constants import PULP_MODULE_ATTR
+
+import gi
+gi.require_version('Modulemd', '2.0')
+from gi.repository import Modulemd as mmdlib  # noqa: E402
+
+
+def _create_snippet(snippet_string):
+    """Create snippet of modulemd[-defaults] as artifact."""
+    tmp_file = tempfile.NamedTemporaryFile()
+    with open(tmp_file.name, 'w') as f:
+        f.write(snippet_string)
+    artifact = Artifact.init_and_validate(tmp_file.name)
+    try:
+        artifact.save()
+    except IntegrityError:
+        artifact = Artifact.objects.get(sha256=artifact.sha256)
+    return artifact
+
+
+def get_modulemd_names(modulemd_yaml, module_index):
+    """Get modulemd names."""
```

Comment:
Arguments need a description in the docstring.

Reply:
more descriptive docstring added to used function over all modulemd.py